views/login.html

```diff
@@ -1,6 +1,6 @@
 {% extends "layout.html" %}
 {% from "input/macro.njk" import govukInput %}
-
+{% set pageHeader = 'Login' %} 
```

Comment:
Shouldn't we use "i18n.login.header" instead of hardcoding pageHeader?

Reply:
@t0nin0s  this is a Idam stub page it doesn't have a page title, it's not going to live